ipn/ipnlocal/local.go

```diff
@@ -2628,6 +2631,18 @@ func (b *LocalBackend) checkExitNodePrefsLocked(p *ipn.Prefs) error {
 	return nil
 }
 
+func (b *LocalBackend) checkFunnelEnabledLocked(p *ipn.Prefs) error {
+	sc0 := b.serveConfig
+	if !sc0.Valid() {
+		return nil
+	}
+	sc1 := sc0.AsStruct()
```

Comment:
Oh thats an expensive call just to check if funnel is enabled. You can selectively expose methods from structs to their corresponding views, so we should add a method "IsFunnelOn" to the view and use it here.

See tailcfg.Node.IsTagged as an example.

Reply:
Updated. PTAL. IsTagged looked pretty standard, but I think I did what we wanted based on other view methods.

Reply:
[posterity] Example @ `dbx.Node.IsTagged`

Updated. PTAL